zipline/pipeline/factors/technical.py

```diff
@@ -598,7 +598,7 @@ class IchimokuKinkoHyo(CustomFactor):
         'kijun_sen_length': 26,
         'chikou_span_length': 26,
     }
-    inputs = USEquityPricing.high, USEquityPricing.close
+    inputs = [USEquityPricing.high, USEquityPricing.low, USEquityPricing.close]
```

Comment:
Thanks @luca-s , makes sense to me! Could I ask for two improvements?

1. Would you make this a tuple of inputs instead of a list, so that no one can mistakenly modify this class-level attribute?
1. Would you add a simple test method to `tests.pipeline.test_technical.IchimokuKinkoHyoTestCase` which ensures that this factor doesn't blow up on computing with the default inputs, as you were seeing?